app/routes/surveys/components/SurveyEditor/SurveyEditor.tsx

```diff
@@ -61,7 +58,7 @@ function TemplateTypeDropdownItems({
 
   return (
     <Dropdown.List>
-      {Object.keys(EVENT_CONSTANTS).map((eventType) => (
+      {Object.keys(EVENT_CONSTANTS).map((eventType: EventType) => (
```

Comment:
wanna make this an enum? :grimacing:

Reply:
Kind of out of scope though, `EVENT_CONSTANTS` is used a lot of places.

Reply:
Sure